locales/en-US/messages.properties

```diff
@@ -88,19 +88,3 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
 
 # Site title when shared on Facebook
 support_mozilla=Support Mozilla
-
-
-# Legacy strings
-first_name=First name
-last_name=Last name
-address=Address
-city=City
-country=Country
-postal_code=Zip/Postal code
-state_province=State/Province
```

Comment:
I think these are all dead.

All of our address components are pretty much dead, but we still have a legacy test we've never run that uses country drop down on the signup page: https://donate.mozilla.org/en-US/thank-you/?test=country-signup-test

I want to kill this dead, but havn't had the chance. I think if we pull these strings out, we should  consider to fixup or remove some of the code that uses it.